config/config.go

```diff
@@ -55,25 +54,25 @@ func ChainlinkVals() map[string]interface{} {
 	return values
 }
 
-// GethNetworks builds the proper geth network settings to use based on the selected_networks config
-func GethNetworks() []environment.SimulatedNetwork {
-	if ProjectConfig.NetworksConfig == nil {
-		log.Error().Msg("ProjectNetworkSettings not set!")
-		return nil
-	}
-	var gethNetworks []environment.SimulatedNetwork
-	for _, network := range ProjectConfig.NetworksConfig.SelectedNetworks {
-		switch network {
-		case DefaultGeth:
-			gethNetworks = append(gethNetworks, environment.DefaultGeth)
-		case PerformanceGeth:
-			gethNetworks = append(gethNetworks, environment.PerformanceGeth)
-		case RealisticGeth:
-			gethNetworks = append(gethNetworks, environment.RealisticGeth)
-		}
-	}
-	return gethNetworks
-}
+//// GethNetworks builds the proper geth network settings to use based on the selected_networks config
```

Comment:
So we're no longer going to pick up different geth types to launch?

Reply:
See comment about multinetwork preset